docs/tsg.md

```diff
@@ -231,6 +231,27 @@ Client connections can drop under various circumstances:
 2. Check app server side event log to see if the app server restarted
 3. Create an issue to us providing time frame, and email the resource name to us
 
+<a name="client_connection_increases_constantly"></a>
+## Client connection increase constanly
+It might caused by improper usage of client connection. 
+
+This might caused by using SignalR clients in Azure Function. If the SignalR client is created and starts in a function method and never stops, then each request will establish SignalR client connection and keep it open. All this connections drops only after the Azure Function or Azure SignalR service  restarts.
```

Comment:
Let's make this more general. No matter whether he is using Azure function, he will meet this problem if he doesn't dispose SignalR client properly.